src/components/QuickLinksDialog/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+
+import GordonDialog from '@material-ui/core/Dialog';
+import MuiDialogTitle from '@material-ui/core/DialogTitle';
+import MuiDialogContent from '@material-ui/core/DialogContent';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+import PropTypes from 'prop-types';
+
+import GordonLinksList from './components/LinksList';
+import './quicklinksdialog.css';
+
+const styles = theme => ({
```

Comment:
I would see if you can create a .scss file and make "root" and "button" entries in the style list. This would follow a consistent format with the rest of the code.

Reply:
Oh, my bad. You have a .scss already! Can root and button be entries on the style sheet or are they something different?

Reply:
Root and button are the simple ones that can stay on the original theme and I just added scss for future style reformation.